src/bindActionCreators.js

```diff
@@ -26,6 +26,11 @@ function bindActionCreator(actionCreator, dispatch) {
  * function.
  */
 export default function bindActionCreators(actionCreators, dispatch) {
+  // return a curried function if only actionCreators are present
+  if(arguments.length === 1) {
+    return (dispatch) => bindActionCreators(actionCreators, dispatch);
```

Comment:
Nit: With a single argument arrow function, the parens are optional. I normally leave them off. Also, we don't use semi colors, so you can drop that.